compose.yaml

```diff
@@ -38,16 +38,13 @@ services:
     image: redis:7-alpine
     restart: unless-stopped
   postgres:
-    image: pgautoupgrade/pgautoupgrade:latest
+    image: postgres:16-alpine
```

Comment:
Absolutely not.

For ARM64, we can use the `pgautoupgrade/pgautoupgrade:16-dev-arm64` tag until I get multiarch builds figured out for the pgautoupgrade project.

Reply:
if i put `pgautoupgrade/pgautoupgrade:16-dev-arm64` image there it'll break setup on amd64. `pgautoupgrade/pgautoupgrade:latest` should be a tag which references both arm64 and amd64 images

Reply:
what benefits are we getting from pgautoupgrade image?

Reply:
> if i put pgautoupgrade/pgautoupgrade:16-dev-arm64 image there it'll break setup on amd64.

Yep, that's why we don't yet claim to support ARM64.

> what benefits are we getting from pgautoupgrade image?

It automatically upgrades the version of PostgreSQL people are using without them needing to do it manually themselves.

It's so people don't get in the situation of being stuck on an ancient, unsupported version of PostgreSQL, as has happened previously with this project.

Reply:
this postgres container is for developers, users are using either setup in getredash/setup or helm chart

Reply:
That's nice.

Reply:
Thinking about this more, I guess we can revert this to the standard official PG image for now.  When the pgautoupgrade one is multi-arch, then we can update this again and it'll upgrade smoothly anyway.